cmake/scripts/common/AddonHelpers.cmake

```diff
@@ -68,6 +68,19 @@ macro (build_addon target prefix libs)
 
   # if there's an addon.xml.in we need to generate the addon.xml
   if(EXISTS ${PROJECT_SOURCE_DIR}/${target}/addon.xml.in)
+    # Set defines used on addon.xml.in and readed from versions.h
+    # To set versions about add-on parts automatic
```

Comment:
"Set defines used in addon.xml.in and read from versions.h to set add-on version parts automatically" ?